Makefile

```diff
@@ -62,28 +83,22 @@ prefclean:
 prere:
 	$(PREFIX)cd libft && make re
 
-clean:
-	$(PREFIX)rm -f $(OBJS) $(ENDPOINT_OBJ) $(VANIA_ENDPOINT_OBJ) $(TANIA_ENDPOINT_OBJ)
-	$(PREFIX)rm -f $(OBJS) $(ENDPOINT_OBJ) $(VANIA_ENDPOINT_OBJ) $(TANIA_ENDPOINT_OBJ)
+clean: testclean
+	$(PREFIX)rm -f $(OBJS) $(ENDPOINT_OBJ) $(TANIA_ENDPOINT_OBJ)
+	$(PREFIX)@if [ -d $(OBJ_F) ]; then $(PREFIX)rm -rf $(OBJ_F); fi
 
 fclean: clean
 	$(PREFIX)rm -f $(NAME)
-	$(PREFIX)rm -f $(ENDPOINT_OBJ)
-	$(PREFIX)rm -f $(ENDPOINT_OBJ)
+	$(PREFIX)rm -f $(TEST_FNAME) $(TEST_TOOL_FNAME)
```

Comment:
This should be not here, but in testfclean, and testfclean should be marked in line 90 as a dependency. Do not duplicate code.

Reply:
https://github.com/ioannnikulin/minishell/pull/72/commits/7dab92bce585b2215ea01e84032c9525a6ac2c43